README.md

```diff
@@ -71,7 +72,54 @@ const client = initialize('00000000-1111-2222-3333-444444444444', {
 })
 ```
 
-### Listening to events from the `client` instance.
+
+## Streaming and Polling Mode
+
+By default, Harness Feature Flags SDK has streaming enabled and polling disabled. Both modes can be toggled according to your preference using the SDK's configurations.
```

Comment:
Generally I'd use `configuration` instead of `configurations`. The configuration is a single entity which contains multiple configuration items.

Reply:
Updated